lib/modules/manager/regex/readme.md

```diff
@@ -13,21 +13,24 @@ The first two required fields are `fileMatch` and `matchStrings`. `fileMatch` wo
 
 Configuration-wise, it works like this:
 
 - You must capture the `currentValue` of the dependency in a named capture group
-- You must have either a `depName` capture group or a `depNameTemplate` config field
-- You can optionally have a `packageName` capture group or a `packageNameTemplate` if it differs from `depName`
-- You must have either a `datasource` capture group or a `datasourceTemplate` config field
-- You can optionally have a `depType` capture group or a `depTypeTemplate` config field
-- You can optionally have a `versioning` capture group or a `versioningTemplate` config field. If neither are present, `semver` will be used as the default
-- You can optionally have an `extractVersion` capture group or an `extractVersionTemplate` config field
-- You can optionally have a `currentDigest` capture group.
-- You can optionally have a `registryUrl` capture group or a `registryUrlTemplate` config field
-  - If it's a valid URL, it will be converted to the `registryUrls` field as a single-length array.
+- You must have either a `depName` capture group or a [`depNameTemplate` config field](https://docs.renovatebot.com/configuration-options/#depnametemplate)
+- You can optionally have a `packageName` capture group or a [`packageNameTemplate` config field] if it differs from `depName`
+- You must have either a [`datasource` capture group](https://docs.renovatebot.com/modules/datasource/#supported-datasources) or a [`datasourceTemplate` config field](https://docs.renovatebot.com/configuration-options/#datasourcetemplate)
+- You can optionally have a `depType` capture group or a [`depTypeTemplate` config field](https://docs.renovatebot.com/configuration-options/#deptypetemplate)  
+  See [manager specific documentation pages](https://docs.renovatebot.com/modules/manager/#supported-managers) for recommended values, other values are also possible.
+- You can optionally have a [`versioning` capture group](https://docs.renovatebot.com/modules/versioning/#supported-versioning) or a [`versioningTemplate` config field](https://docs.renovatebot.com/configuration-options/#versioningtemplate). If neither are present, `semver` will be used as the default
+- You can optionally have an `extractVersion` capture group or an [`extractVersionTemplate` config field](https://docs.renovatebot.com/configuration-options/#extractversiontemplate)
+- You can optionally have a [`currentDigest` capture group](https://docs.renovatebot.com/configuration-options/#digest).
+- You can optionally have a [`registryUrl` capture group](https://docs.renovatebot.com/configuration-options/#registryurls) or a [`registryUrlTemplate` config field](https://docs.renovatebot.com/configuration-options/#registryurltemplate)
+  - If it's a valid URL, it will be converted to the [`registryUrls` field](https://docs.renovatebot.com/configuration-options/#registryurls) as a single-length array.
```

Comment:
```suggestion
- You can optionally have a [`registryUrl` capture group](https://docs.renovatebot.com/configuration-options/#registryurls) or a [`registryUrlTemplate` config field](https://docs.renovatebot.com/configuration-options/#registryurltemplate), if it's a valid URL, it will be converted to the [`registryUrls` field](https://docs.renovatebot.com/configuration-options/#registryurls) as a single-length array
```

MkDocs or Material for MkDocs doesn't know how render a list like this:

- You can optionally have a registryUrl capture group or a registryUrlTemplate config field
  - If it's a valid URL, it will be converted to the registryUrls field as a single-length array.
  
The list is also broken on the published [`regex` manager page](https://docs.renovatebot.com/modules/manager/regex/#required-fields).

Let's put the `if` part on the same line, so it renders properly.

Reply:
https://github.com/renovatebot/renovate/pull/15067/commits/29200839ba01c0e61d8481436b27d4ac04d084c2

Reply:
please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting to reduce noise

Reply:
🤔 you still get an email about resolution, and also how do you take a look if I had done the right thing? (There's a suggestion, but I might've not done it exactly like that. For example in this particular comment I didn't join two sentences with a comma, I kept the sentences instead, because otherwise it becomes a run-on sentence.) Anyway, I'll press the button without linking the relevant changes in the future if that helps.

Reply:
We review the new changes and do new comments if required. We never review single commits. So `done` or `<git-sha>` comments  are useless for us.

Of cause you should comment, if something needs further discussion. So don't resolve without any code change or comment, 😉 

Also github has a nice option to batch suggestions to a single commit on the review page  😉

Reply:
Ah, ok, good faith and end result matters principles 😅. Thanks for the clarification!